microsoft-edge/webview2/release-notes.md

```diff
@@ -42,6 +42,228 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
+   * [ICoreWebView2EnvironmentOptions4::SetCustomSchemeRegistrations method](/microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions4?view=webview2-1.0.1587.40&preserve-view=true#setcustomschemeregistrations)
+
+---
+
+
+<!-- ====================================================================== -->
+## 1.0.1671-prerelease
+
+Release Date: February 15, 2023  
+
+[NuGet package for WebView2 SDK 1.0.1671-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1671-prerelease)
+
+For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 112.0.1671.0 or higher. 
+
+### General
+
+#### Experimental features 
+
+*  Added support for the Experimental File API:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2ExperimentalFile Class](/dotnet/api/microsoft.web.webview2.core.corewebview2experimentalfile?view=webview2-dotnet-1.0.1671-prerelease&preserve-view=true)
+   * [CoreWebView2ExperimentalFile.Path Property](/dotnet/api/microsoft.web.webview2.core.corewebview2experimentalfile.path?view=webview2-dotnet-1.0.1671-prerelease&preserve-view=true)
+* [CoreWebView2ExperimentalWebMessageReceivedEventArgs Class](/dotnet/api/microsoft.web.webview2.core.corewebview2experimentalwebmessagereceivedeventargs?view=webview2-dotnet-1.0.1671-prerelease&preserve-view=true)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2ExperimentalFile Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2experimentalfile?view=webview2-winrt-1.0.1671-prerelease&preserve-view=true)
+   * [CoreWebView2ExperimentalFile.Path Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2experimentalfile?view=webview2-winrt-1.0.1671-prerelease&preserve-view=true#path)
+* [CoreWebView2ExperimentalWebMessageReceivedEventArgs Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2experimentalwebmessagereceivedeventargs?view=webview2-winrt-1.0.1671-prerelease&preserve-view=true)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2ExperimentalFile interface](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalfile?view=webview2-1.0.1671-prerelease&preserve-view=true)
+   * [ICoreWebView2ExperimentalFile::get_Path method](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalfile?view=webview2-1.0.1671-prerelease&preserve-view=true#get_path)
+* [ICoreWebView2ExperimentalWebMessageReceivedEventArgs interface](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalwebmessagereceivedeventargs?view=webview2-1.0.1671-prerelease&preserve-view=true)
+
+---
+
+*  Added support for Experimental Object Collection View API:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2ExperimentalObjectCollectionView Class](/dotnet/api/microsoft.web.webview2.core.corewebview2experimentalobjectcollectionview?view=webview2-dotnet-1.0.1671-prerelease&preserve-view=true)

```

Comment:
@peiche-jessica - delete these 3 links?

Reply:
No CoreWebView2ExperimentalObjectCollectionView in .NET nor WinRT. Please delete them.

Reply:
del'd them locally